services/ttl.js

```diff
@@ -0,0 +1,32 @@
+const config = require('../config');
+const {DatabaseConnector} = require("../crdc-datahub-database-drivers/database-connector");
+const {MongoDBCollection} = require("../crdc-datahub-database-drivers/mongodb-collection");
+const {DATABASE_NAME, SESSION_COLLECTION} = require("../crdc-datahub-database-drivers/database-constants");
+const dbConnector = new DatabaseConnector(config.mongo_db_connection_string);
+
+
+
+
+const getTTL = (req, res) => {
+    const sessionID = req.sessionID
+    if (sessionID){
```

Comment:
Why did you declare sessionID?

Reply:
I just though I might check if there is sessions before going through.

Reply:
You should not declare another variable. I dont see sessionID reused in this function.